batch_write_items.go

```diff
@@ -40,5 +48,21 @@ func (t *Client) BatchWriteItems(ctx context.Context, input []map[string]types.A
 	}
 
 	return nil
+}
+
+func (t *Client) BatchPutItems(ctx context.Context, inputs []*BatchPutItemsInput) error {
```

Comment:
```suggestion
func (t *Client) BatchPutItems(ctx context.Context, inputs []BatchPutItemsInput) error {
```

Reply:
The slice is already a pointer, it doesn't make any difference when passing arguments.